shim/opentelemetry-opencensus-shim/src/opentelemetry/shim/opencensus/__init__.py

```diff
@@ -22,3 +22,16 @@
 already instrumented using OpenCensus to start using OpenTelemetry with minimal effort, without
 having to rewrite large portions of the codebase.
 """
+
+from opentelemetry.shim.opencensus._patch import install_shim, uninstall_shim
+
+__all__ = [
+    "install_shim",
+    "uninstall_shim",
+]
+
+# TODO: Decide when this should be called.
```

Comment:
Another option discussed in the SIG is to add to `opentelemetry-api` (at module level) a `for` loop that goes through a set of newly-defined entry points that run some functions. In this way, we could register this or any other shim into those entry points so that they can do their thing automatically even when not using `opentelemetry-instrument`.

Reply:
> `install_shim` would add a public API function, right?

Yes but this will be exposed either way. Also this package will not be marked stable yet (the spec is not marked stable). I think we can leave it as is and open an issue to discuss.